main.py

```diff
@@ -437,4 +437,9 @@ def main():
                                    |_|      |___/ 
 """)
     print("Automatische Terminbuchung für den Corona Impfterminservice\n")
+
+    print("Vor der Ausführung des Programms ist die Berechtigung zur Impfung zu prüfen.\n"
+          "Ob Anspruch auf eine Impfung besteht, kann hier nachgelesen werden:\n"
```

Comment:
Bitte hier noch einmal explizit erwähnen, dass die Priorisierung für J&J bei Impfzentren nicht aufgehoben ist. Da war nämlich abweichendes in den Medien, das sich aber bisher entweder nur auf Hausärzte bezieht, oder noch nicht in Kraft ist. (Zumindest nicht in BW. Andere Bundesländer sollten noch mal gecheckt werden.)

Reply:
In circa 1,5 Wochen wird dann auch die Prioritisierung aufgehoben und jeder darf. Dann sollte man nicht vergessen den Hinweis wieder rauszunehmen, oder man macht das print statement vom aktuellen datum abhängig. 
Sth. along the lines: 
if today() < 7.6.2021 print(hinweis prio 3) 

Das sollte dann allerdings trotzdem entfernt werden, wenn der Tag gekommen ist

Reply:
Belassen wir es bei der allgemeinen Formulierung, dass der Anspruch zu prüfen ist. Auch wenn die Prio irgendwann aufgehoben wird, schadet der Hinweis nicht.